smartrouter/image.yaml

```diff
@@ -74,6 +74,12 @@ envs:
     - name: "KIE_SERVER_CONTROLLER_TOKEN"
       example: "dasd373egds"
       description: "KIE server controller token for bearer authentication (Sets the org.kie.server.controller.token system property)"
+    - name: "LOG_LEVEL"
+      example: "INFO"
```

Comment:
change the example to DEBUG and in the description add note that if empty it defaults to info

Reply:
ok